styles/index.css

```diff
@@ -41,8 +41,8 @@ body {
   grid-template-rows: auto 1fr;
```

Comment:
The saber overlaps with the quiz on tablets and smaller screens. To keep it always below the quiz, you could set `grid-template-rows` to something like `1fr auto 0.2fr` and remove `position: relative` and `top` properties from `#lightSabreProgresBar`. Consider also changing the height of the `img` to make the saber and the time counter closer to the quiz.